Makefile.core.mk

```diff
@@ -280,7 +280,8 @@ BINARIES:=./istioctl/cmd/istioctl \
   ./pkg/test/echo/cmd/server \
   ./mixer/test/policybackend \
   ./operator/cmd/operator \
-  ./cni/cmd/istio-cni ./cni/cmd/istio-cni-repair
+  ./cni/cmd/istio-cni ./cni/cmd/istio-cni-repair \
+  ./tools/istio-iptables
```

Comment:
hmm.. right now this is part of proxyv2, I guess the main() file is just left over. Its only 3 lines long though so maybe not a big concern